### PR TITLE
fix(scheme-langserver):defaultly enable log/out/multi-thread and disable type-inference

### PR DESCRIPTION
**Pull request by ufo5260987423**

### CODE REVIEW

File: lua/lspconfig/server_configurations/scheme_langserver.lua [hunk 1]
```diff
@@ -1,6 +1,6 @@
 local util = require 'lspconfig.util'
 local bin_name = 'scheme-langserver'
-local cmd = { bin_name }
+local cmd = { bin_name, "~/.scheme-langserver.log", "enable", "disable" }
```
Comment (glepnir) [inline lua/lspconfig/server_configurations/scheme_langserver.lua:3]: single quote

File: lua/lspconfig/server_configurations/scheme_langserver.lua [hunk 2]
```diff
@@ -1,6 +1,6 @@
 local util = require 'lspconfig.util'
 local bin_name = 'scheme-langserver'
-local cmd = { bin_name }
+local cmd = { bin_name, '~/.scheme-langserver.log', 'enable', 'disable' }
```
Comment (glepnir) [inline lua/lspconfig/server_configurations/scheme_langserver.lua:3]:
```suggestion
local cmd = { 'scheme-langserver', '~/.scheme-langserver.log', 'enable', 'disable' }
```
remove bin_name

File: lua/lspconfig/server_configurations/scheme_langserver.lua (hunk 1 above)
Comment (ufo5260987423) [reply lua/lspconfig/server_configurations/scheme_langserver.lua]: OK, I've updated.

File: lua/lspconfig/server_configurations/scheme_langserver.lua (hunk 2 above)
Comment (ufo5260987423) [reply lua/lspconfig/server_configurations/scheme_langserver.lua]: ok

**Pull request merged**